SxGDriveTokenGenerator.py

```diff
@@ -210,22 +206,13 @@ def main():
         print("Locating client secret file")
         client_secret_path = config.find_client_secret_path()
 
-        # Create an empty token file so oath2client doesn't print a warning on console
-        create_empty_file(config.get_output_token_json_path())
-
         print("Performing authentication")
         do_auth(config.get_output_token_json_path(), client_secret_path, config.scopes)
 
-        print("Copying client secret JSON to output directory")
-        shutil.copyfile(client_secret_path, config.get_output_client_secret_json_path())
-
-        input(f"\nProcess complete! Copy (2) files from " +
+        input(f"\nProcess complete! Copy (1) file from " +
               f"'{str(Path(config.output_folder_path).absolute())}' to your Switch SD card " +
               f"folder '/switch/sx/'. Press [Enter] to close this window.")
 
-    except shutil.Error as shutil_ex:

```

Comment:
Removed this as this error seems to be related to copying client secret file, which is not needed anymore.